tests/EventStore/Tests/StreamFeed/EntryTest.php

```diff
@@ -11,4 +11,16 @@ public function testTypeGetter()
 
         $this->assertEquals('Bar', $entry->getType());
     }
-}
+
+    /**
+     * @test
+     */
+    public function theEventTypeForEntriesOfAnProjectionStreamFeedIsCorrect(){
+        $json = file_get_contents(__DIR__ . '/Fixtures/RichEventStoreCeEntryResponse.json');
+        $entryData = json_decode($json, TRUE);
```

Comment:
please run php-cs-fixer to fix coding standard (if you're using Linux just copy hooks/pre-commit in .git/hooks)